webapp-admin/src/components/SystemUserForm.js

```diff
@@ -100,17 +101,30 @@ class SystemUserForm extends Component {
         this.setState({until: date});
     }
 
+    handleAddEmptySerialField = (e) => {
+        e.preventDefault()
+        this.setState({serials: this.state.serials.concat([''])})
```

Comment:
I would suggest creating a `tmpSerials` simular to `handleAddSerialNumber` and just push an empty string to it:
```js
let tmpSerials = this.state.serials;
tmpSerials.push('');
this.setState({serials: tmpSerials});  
```